packages/runtime/src/internal/RendererContext.ts

```
@@ -443,6 +452,11 @@ export class RendererContext {
     }
     node.sibling = oldNode.sibling;
 
+    // Clean up old node
+    delete oldNode.child;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)